src/openrct2-ui/windows/Themes.cpp

```diff
@@ -829,6 +873,56 @@ static WindowClass window_themes_tab_7_classes[] = {
             return classes[index];
         }
 
+        int8_t GetColourIndex(int32_t y)
+        {
+            int8_t total = 0;
```

Comment:
This seems to count the number of rows (occupied or not). In that case, `numRows` seems like a more appropriate name. This comment also applies to other cases where you count the number of rows.